src/rest-server/src/models/v2/virtual-cluster/k8s.js

```diff
@@ -19,23 +19,157 @@
 const util = require('util');
 const createError = require('@pai/utils/error');
 const vcConfig = require('@pai/config/vc');
+const k8s = require('@pai/utils/k8sUtils');
+const axios = require('axios');
+const yaml = require('js-yaml');
 
 
 class VirtualCluster {
   constructor() {
```

Comment:
write separate functions instead of a class in this file

Reply:
Done